### ISSUE TITLE
NVDA is reading page title instead of reading the popup title or nothing.

### ISSUE THREAD

**Issue by harishreddykanakanaka** [nvaccess/nvda#9047]
### Steps to reproduce:
1) Open Test2.html attached below.
2)Tab till Save/ Delete button and hit Enter. 
3)Verify how NVDA is reading the popup

### Actual behavior:
NVDA is reading page title instead of reading the popup title for publish and delete popup
### Expected behavior:
NVDA should read the popup title(This site says) but reading nothing.
### System configuration 
windows 10 64bit ,16GB ram
#### NVDA version:
2018.2.1
#### Windows version:
Windows 10 Enterprise, version 1803
#### Other information about your system:
 Latest version of chrome.
#### Have you tried any other versions of NVDA?
No

**Importance of experience (​​​​​High or Low Impact):**
It will create high impact on user as user doesn't know whether popup opened or not.

**Quality of Experience (High or Low Impact):**
Quality of experience will be low.

**Any alternate solutions or technologies evaluated**:
Working fine with Narrator and Android Talk-back.

 evidence :
[test2.zip](https://github.com/nvaccess/nvda/files/2667426/test2.zip)

**Comment by Adriani90** [comment]
@harishreddykanakanaka, I am testing with NVDA 2018.4 RC1 on Google Chrome 71. I cannto reproduce it. When I press enter on save out of focus mode, it reads the notification "do you want to save?". It does not read the page title. When pressing enter from the browse mode, also the notification is read but not the title. I can enable browse mode after pressing enter on save or delete and then I am imediately in the popup area.

**Comment by harishreddykanakanaka** [comment]
@Adriani90  That's the alert message. It should read This site says then Ok/cancel button.
The same is read by Narrator in Edge.

**Comment by harishreddykanakanaka** [comment]
> @harishreddykanakanaka, I am testing with NVDA 2018.4 RC1 on Google Chrome 71. I cannto reproduce it. When I press enter on save out of focus mode, it reads the notification "do you want to save?". It does not read the page title. When pressing enter from the browse mode, also the notification is read but not the title. I can enable browse mode after pressing enter on save or delete and then I am imediately in the popup area.

**Comment by harishreddykanakanaka** [comment]
@Adriani90  The bug is **It should read This site says and then ok/cancel button**. 
The same is read by Narrator in Edge browser please check and verify the difference.

**Comment by Adriani90** [comment]
This is fixed in NVDA alpha-20015,56a62097 and Crome 81.
Closing as works for me.
